cmd/erasure-multipart.go

```diff
@@ -381,7 +381,7 @@ func (er erasureObjects) newMultipartUpload(ctx context.Context, bucket string,
 		rctx := lkctx.Context()
 		obj, err := er.getObjectInfo(rctx, bucket, object, opts)
 		lk.RUnlock(lkctx)
-		if err != nil && !isErrVersionNotFound(err) {
+		if err != nil && !isErrVersionNotFound(err) && !isErrObjectNotFound(err) {
```

Comment:
Isn't version id always passed here ?

Reply:
if version is null, it will return ObjectNotFound err

Reply:
so this issue is for existing object replication from unversioned to versioned buckets?

Reply:
yes